draftlogs/.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
Did you mean this file to be `.gitkeep`? I guess there's a [competing convention to use `.gitignore`](https://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep) but I agree with the sentiment that this is more confusing than `.gitkeep`.

Reply:
Yes. I could change that if you want.

Reply:
Either .gitkeep, or a README explaining these files - actually a README might be better, then the PR template could just refer to it, but when contributors are actually writing these files they'll have the README to look at.

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/5780/commits/3a91c0e380886d6292b12a71b22fd25d298b5f82.